packages/ui-components/src/Tabs/styles.scss

```diff
@@ -238,3 +238,14 @@ button, .btn, .nav-link {
 .btn {
   transition: none !important;
 }
+
+.custom-drag-layer {
+  .custom-drag-layer-box1 {
```

Comment:
should use more clear class name, eg: `.custom-drag-layer-original`, `.custom-drag-layer-copy`. BTW, you should have a double-check if there should use `rem` or `px`